manager/docker_manager.go

```diff
@@ -0,0 +1,1181 @@
+// Copyright 2014 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Manager of cAdvisor-monitored containers.
+package manager
```

Comment:
I feel like this should be in a separate package (`docker`). That will make it easier to match the plugin style used elsewhere (storage), and help to keep runtimes dependencies isolated.